frontend/app/src/components/ToolbarActions/ToolbarActions.tsx

```diff
@@ -72,12 +76,15 @@ function ToolbarActions({
           key={key}
           label={label}
           icon={icon}
-          onClick={() =>
+          onClick={() => {
+            metricsMgr.enqueue("menuClick", {
```

Comment:
Should this be toolbarClick?

Reply:
The name is a bit historical, but we decided to unify all click events under this event name. This makes it a lot easier to manage the data and build more universal metric apps.

Reply:
It would be good to rename the event name. But it would not be worth the effort thats required to support a new event